common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -275,7 +278,9 @@ function(_) {
                     self.playerState = PlayerState.PLAYING;
                     self.playVideo();
                 }
-            });
+            };
+            this.videoEl.on('click', callback);
+            this.el.find('.video-wrapper .btn-play').on('click', callback);
```

Comment:
can we declare it on one place and use it where ever needed?
`this.videoOverlayEl = this.el.find('.video-wrapper .btn-play');`
it will be faster. – https://jsperf.com/scoped-unscoped-selectors/1